static/app/components/profiling/threadSelector.tsx

```diff
@@ -58,7 +72,21 @@ function ThreadMenuSelector({
         icon: <IconList size="xs" />,
         size: 'xs',
       }}
-      options={options}
+      options={[
+        {
+          // TODO: fix CompactSelect types to better represent usage.
+          // this value has no effect on the selection, it's simply to satisfy the types..
+          // tried modifying the types but it creates a lot of errors
+          value: threadId || 0,
+          label: t('Profiles'),
+          options: profileOptions,
+        },
```

Comment:
tsc errors when modifying `options` to support `Array<{label:string; options: OptionType[]}>`
https://github.com/getsentry/sentry/actions/runs/3491715766/jobs/5844677026